src/shared/sam/cli/samCliValidator.ts

```diff
@@ -9,7 +9,7 @@ import { SamCliConfiguration } from './samCliConfiguration'
 import { SamCliInfoInvocation, SamCliInfoResponse } from './samCliInfo'
 import { SamCliProcessInvoker } from './samCliInvokerUtils'
 
-export const MINIMUM_SAM_CLI_VERSION_INCLUSIVE = '0.32.0'
+export const MINIMUM_SAM_CLI_VERSION_INCLUSIVE = '0.38.0'
```

Comment:
Please make a new (separate) changelog entry mentioning the minimum sam version has changed. See the changelog file for what this looks like.

Reply:
Sure, would this be a good description -  Minimum version of SAM CLI has been adjusted from 0.32.0 to 0.38.0 to accommodate new SAM application support for EventBridge Schemas